desktop/dev-environments.md

```diff
@@ -2,8 +2,7 @@
 description: Dev Environments
 keywords: Dev Environments, share, collaborate, local
 title: Development Environments Preview
-toc_min: 1
-toc_max: 2
+
 ---
 
 Dev Environments enable you to collaborate easily by allowing you to share work-in-progress code with your team members. When using Dev Environments, you can easily set up repeatable development environments, keeping the environment details versioned along with your code. You can also share your work-in-progress code with your team members in just one click and without having to deal with any merge conflicts while moving between Git branches to get your code on to their machine.
```

Comment:
Minor suggestion: 
Dev Environments lets you collaborate easily by allowing you to share work-in-progress code with your team members. When using Dev Environments, you can easily set up repeatable development environments, keeping the environment details versioned along with your code. You can also share your work-in-progress code with your team members in just one click and without dealing with any merge conflicts while moving between Git branches to get your code onto their machine.